EssentialCSharp.Web/EssentialCSharp.Web.csproj

```diff
@@ -1,32 +1,56 @@
-<Project Sdk="Microsoft.NET.Sdk.Web">
-  <PropertyGroup>
-    <TargetFramework>net7.0</TargetFramework>
-    <AccessToNugetFeed>true</AccessToNugetFeed>
-    <UserSecretsId>18e91e0d-ea2d-490f-b77e-ec008f9d09ec</UserSecretsId>
-  </PropertyGroup>
-  <ItemGroup>
-    <PlaceholderHtmlFile Include="$(ProjectDir)Placeholders/*.html" />
-    <PlaceholderJsonFile Include="$(ProjectDir)Placeholders/*.json" />
-  </ItemGroup>
-  <ItemGroup Condition="$(AccessToNugetFeed)">
-    <PackageReference Include="ContentFeedNuget" Version="1.1.0-3348" />
-  </ItemGroup>
-  <ItemGroup>
-    <PackageReference Include="HtmlAgilityPack" Version="1.11.54" />
-    <PackageReference Include="IntelliTect.Multitool" Version="1.4.0" />
-    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="7.0.11" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.RuntimeCompilation" Version="7.0.13" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.19.5" />
-  </ItemGroup>
-  <ItemGroup>
-    <Content Update="wwwroot\images\00mindmap.svg">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="Chapters\**\*.html" CopyToPublishDirectory="PreserveNewest" />
-  </ItemGroup>
-  <Target Condition="!$(AccessToNugetFeed)" Name="CopyPlaceholderContent" BeforeTargets="Build">
-    <Copy SourceFiles="@(PlaceholderJsonFile)" DestinationFolder="$(ProjectDir)/Chapters/" SkipUnchangedFiles="true" />
-    <Copy SourceFiles="@(PlaceholderHtmlFile)" DestinationFolder="$(ProjectDir)/Chapters/01/Pages/" SkipUnchangedFiles="true" />
-  </Target>
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+    <PropertyGroup>
+        <TargetFramework>net7.0</TargetFramework>
+        <AccessToNugetFeed>true</AccessToNugetFeed>
+        <UserSecretsId>18e91e0d-ea2d-490f-b77e-ec008f9d09ec</UserSecretsId>
```

Comment:
I would move this to our Directory.Build.props so all projects uses the same secrets.